plugins/jira-dashboard/src/components/JiraUserIssuesCard/JiraUserIssuesCard.tsx

```diff
@@ -0,0 +1,83 @@
+import {
+  BottomLinkProps,
+  InfoCard,
+  Progress,
+  ResponseErrorPanel,
+  TableColumn,
+} from '@backstage/core-components';
+
+import React from 'react';
+import {
+  columnKey,
+  columnPriority,
+  columnStatus,
+  columnSummary,
+  columnUpdated,
+  JiraTable,
+} from '../JiraTable';
+import { useApi } from '@backstage/core-plugin-api';
+import { jiraDashboardApiRef } from '../../api';
+import { useJiraUserIssues } from '../../hooks/useJiraUserIssues';
+import { Issue } from '@axis-backstage/plugin-jira-dashboard-common';
+
+export type JiraUserIssuesCardProps = {
+  title?: string;
+  maxResults?: number;
+  bottomLinkProps?: BottomLinkProps;
+};
+
+export const userColumns: TableColumn<Issue>[] = [
+  columnKey,
+  columnSummary,
+  columnPriority,
+  columnStatus,
+  columnUpdated,
+];
+
+export const JiraUserIssuesCard = ({
+  title = 'My open issues',
+  maxResults = 15,
+  bottomLinkProps,
+}: JiraUserIssuesCardProps) => {
+  const api = useApi(jiraDashboardApiRef);
+
+  const {
+    data: jiraResponse,
+    loading,
+    error,
+  } = useJiraUserIssues(maxResults, api);
+
+  if (loading) {
+    return <Progress />;
+  } else if (error) {
```

Comment:
The "else" can be removed here, since we return if previous "if" is true?

Reply:
Code updated. Note I made just a copy of your existing code in your main branch. See JiraDashboardContent.tsx...